Dockerfile.d/etc_udev_rules.d_90-flannel.rules

```diff
@@ -3,3 +3,4 @@
 # https://github.com/kubernetes/kops/pull/9074
 # https://github.com/karmab/kcli/commit/b1a8eff658d17cf4e28162f0fa2c8b2b10e5ad00
 SUBSYSTEM=="net", ACTION=="add|change|move", ENV{INTERFACE}=="flannel.1", RUN+="/usr/sbin/ethtool -K flannel.1 tx-checksum-ip-generic off"
+SUBSYSTEM=="net", ACTION=="add|change|move", ENV{INTERFACE}=="flannel.1", RUN+="/usr/sbin/ethtool --offload eth0 tx-checksum-ip-generic off"
```

Comment:
Not flannel.1

Reply:
@AkihiroSuda I removed that parameter, but I don't know how the rules work and suspect something else should be there. Can you take a look?

Reply:
Is the interface `eth0` and then we remove `--offload etho` from the RUN?